README.md

```diff
@@ -106,7 +122,24 @@ A little note on terminology:
 * "provisioning" - we use this to mean configuring the server with Ansible.
 * "deployment" - we use to mean installing or updating the web application with Capistrano.
 
-## Current state of this work (as of 26 May 2018)
+## <a name='Updates'></a>Updates 
+### <a name=''></a>2025-05-27
+_Umm. 7 years later, plus one day. That's weird._
+
+#### <a name='SupportedPlatforms'></a>Supported Platforms
+In the past, the tools in this repo were well supported across most common Linux platforms (including WSL), and OS X. However, newer versions of OSX only run on ARM chips, and older versions of OS X are increasingly unsupported by tools such as VirtualBox and Docker.
+
+As of today, the only platform that we know works is debian-based Linux systems. Other linuxes probably work, including WSL; and there are probably two releases of MacOS which still run on the last generations of Intel Macs which might work.
+
+We'd like to expand this in future, when we have time
+
+#### <a name='RightToKnowDevplatform'></a>RightToKnow Dev platform
+We've moved RTK on to upstream Alavateli, so the instructions below for a dev environment are out of date. Please refer to [openaustralia/righttoknow](https://github.com/openaustralia/righttoknow?tab=readme-ov-file#development)'s README for instructions.
+
+#### <a name='PlanningAlertsProduction'></a>PlanningAlerts Production
+We now have two productions servers, and a blue/green deployment process driven out of Terraform
```

Comment:
We still use capistrano for day-to-day deployment of PlanningAlerts (when you're just updating the application). We only use the blue/green deployment when we're doing server updates or major software updates (e.g. updating ruby version or upgrading rails) and we want to make sure that we can update things without downtime.